### PR TITLE
Add period inference from freq strings to sktime

### PR DESCRIPTION
**Pull request by canerturkmen**
*Issue #, if available:*

*Description of changes:*
- Add period inference from freq strings to sktime
- Add `pred_time_test`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1914-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1914/1/index.html

**Review by Innixma — APPROVED** [review]

**Comment by szha** [comment]
Job PR-1914-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1914/2/index.html

**Comment by szha** [comment]
Job PR-1914-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1914/3/index.html

File: timeseries/src/autogluon/timeseries/models/sktime/abstract_sktime.py [hunk 1]
```diff
@@ -69,13 +70,16 @@ def __init__(
         self.skt_forecaster: Optional[BaseForecaster] = None
         self._fit_index: Optional[pd.Index] = None
 
-    def _get_skt_forecaster(self) -> BaseForecaster:
+    def _get_skt_forecaster(self, sp: Optional[int] = None) -> BaseForecaster:
         """Return the sktime forecaster object for the model"""
+        params = self._get_model_params().copy()
+        if sp:
```
[the remark this answers is not in the document]
Comment (canerturkmen) [reply timeseries/src/autogluon/timeseries/models/sktime/abstract_sktime.py:76]: noted. let's see what the benchmark says.

**Pull request merged**